.github/ISSUE_TEMPLATE/01-bug-report.yaml

```diff
@@ -0,0 +1,64 @@
+name: 🐞 Bug report
```

Comment:
I will assume you have tested all of those and they show up well.

Reply:
Yes, you can try them out here if you like https://github.com/taxprofiler/taxpasta/issues/new/choose